lib/web/apiserver.go

```diff
@@ -64,6 +64,11 @@ import (
 	"github.com/gravitational/teleport/lib/secret"
 )
 
+const (
+	loginFailedURL            = "/web/msg/error/login"
```

Comment:
Would be nice to add godocs on these constants explaining difference between them and in which case one is shown vs the other.

Reply:
added comment